pystan/stanc.hpp

```diff
@@ -38,12 +43,15 @@ int stanc(std::string model_stancode, std::string model_name, PyStancResult& res
 
   std::string mcode_ = model_stancode;
   std::string mname_ = model_name;
+  bool aundefined_ = allow_undefined;
+  std::string fname_ = filename; 
+  std::vector<std::string> ipaths_ = include_paths;
```

Comment:
Are any of these definitions necessary?

Reply:
Probably not, I just kept the original structure

Reply:
Let's get rid of them and see if works?